app/core/store/modules/tracker.js

```diff
@@ -2,6 +2,8 @@ export default {
   namespaced: true,
 
   state: {
+    doNotTrack: window.navigator.doNotTrack === "1",
```

Comment:
This is awesome.
Is it worth checking that `navigator` exists?

Does [this site suggest](https://caniuse.com/#feat=mdn-api_navigator) that 62% of users have a navigator object?

Reply:
Fixed!